.circleci/config.yml

```diff
@@ -196,7 +196,7 @@ jobs:
           # to manually avoid the case where bad dependencies
           # are cached, and has no other meaning.
           # If you update it, be sure to update save_cache too.
-          key: velox-circleci-macos-deps-v1-{{ checksum ".circleci/config.yml" }}-{{ checksum "scripts/setup-macos.sh" }}
```

Comment:
Forcing key change here so as to update the cache - which for some reason doesnt seem to get updated even if setup-macos.sh has changed.